docs/concepts/whatisvpp/modularflexibleextensible.rst

```diff
@@ -0,0 +1,43 @@
+.. _modularflexibleextensible:
+
+.. toctree::
+
+Modular, Flexible, and Extensible
+***********************************
+
+The FD.io VPP platform is built on a ‘packet processing graph’. This modular
```

Comment:
The FD.io VPP packet processing pipeline is decomposed into a ‘packet processing graph